user/agate/template.py

```diff
@@ -0,0 +1,17 @@
+pkgname = "agate"
+pkgver = "3.3.14"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo-auditable", "pkgconf"]
+makedepends = ["rust-std"]
+checkdepends = ["openssl3-devel"]
```

Comment:
```suggestion
makedepends = ["openssl3-devel"]
```

Reply:
please stop suggesting incorrect things

Reply:
sorry. it seems i have some gaps in my knowledge.

i'll double-check myself before suggesting something in the future.